data/src/main/java/org/oppia/android/data/backends/gae/api/PlatformParameterService.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.android.data.backends.gae.api
+
+import org.oppia.android.data.backends.gae.model.GaePlatformParameters
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/** Service that provides access to platform parameter endpoint. */
+interface PlatformParameterService {
+
+// TODO("Change the url to point to the correct endpoint when the backend is ready")
+  @GET("platform_features_evaluation_handler")
+  fun getPlatformParametersByVersion(
+    @Query("app_version") version: String
```

Comment:
platform_type also should be populated. It's important for the backend to know the request is originating from Android.

Reply:
Done. 
These are the reference files for the allowed value of platform_type

- [https://github.com/oppia/oppia/blob/b1846a839bfb72a5430031fcd2e27638004ad92b/core/domain/platform_parameter_domain.py#L78](https://github.com/oppia/oppia/blob/b1846a839bfb72a5430031fcd2e27638004ad92b/core/domain/platform_parameter_domain.py#L78)
- [https://github.com/oppia/oppia/blob/b1846a839bfb72a5430031fcd2e27638004ad92b/assets/constants.ts#L5830](https://github.com/oppia/oppia/blob/b1846a839bfb72a5430031fcd2e27638004ad92b/assets/constants.ts#L5830)